riak/tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist = riak, flake8
+
+[testenv]
+platform = linux|darwin|win32
+
+[testenv:riak]
+deps =
+    ../datadog-checks-base
+    -rrequirements-dev.txt
+setenv = REDIS_VERSION=3.2
+passenv = DOCKER_HOST*
+commands =
+    pip install --require-hashes -r requirements.txt
+    pytest -v -r apP --log-level debug
```

Comment:
Do we need all of this output?

Reply:
I don't know. We should discuss with the team. It might be helpful but you're right it's probably too much

Reply:
I think `-v` is nice to have but I agree this might be too much.

Reply:
I'll remove everything bug the -v